eidas_node/tests/data/saml_request.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8' standalone='yes'?>
-<saml2p:AuthnRequest xmlns:saml2="urn:oasis:names:tc:SAML:2.0:assertion" xmlns:saml2p="urn:oasis:names:tc:SAML:2.0:protocol" xmlns:eidas="http://eidas.europa.eu/saml-extensions" xmlns:xmlenc="http://www.w3.org/2001/04/xmlenc#" Consent="urn:oasis:names:tc:SAML:2.0:consent:unspecified" Destination="test/destination" ID="test-saml-request-id" IssueInstant="2017-12-11T14:12:05.148Z" Version="2.0" IsPassive="false" ForceAuthn="true" ProviderName="DEMO-SP-CA">
+<saml2p:AuthnRequest xmlns:saml2="urn:oasis:names:tc:SAML:2.0:assertion" xmlns:saml2p="urn:oasis:names:tc:SAML:2.0:protocol" xmlns:eidas="http://eidas.europa.eu/saml-extensions" Consent="urn:oasis:names:tc:SAML:2.0:consent:unspecified" Destination="test/destination" ID="test-saml-request-id" IssueInstant="2017-12-11T14:12:05.148Z" Version="2.0" IsPassive="false" ForceAuthn="true" ProviderName="DEMO-SP-CA">
```

Comment:
This is a result of the removal of `xmlenc` from `EIDAS_NAMESPACES`. `xmlenc` and `ds` are not in `EIDAS_NAMESPACES` because they are not necessary unless the document is signed/encrypted.